### PR TITLE
Added Irctc and livescore api

### PR DESCRIPTION
**Pull request by anupeshverma**

### CODE REVIEW

**Comment by TheLearneer** [comment]
@anupeshverma please stop making multiple PRs 🙏

**Comment by anupeshverma** [comment]
I have deleted the previous one

**Review by TheLearneer — CHANGES_REQUESTED** [review]
There is no IRCTC API docs, but only the logo??

**Comment by anupeshverma** [comment]
Sorry for the mistake 🥺

File: src/content/apis/sports/LiveScore API.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+name: LiveScore API
+description: This API helps to query for football, cricket, basketball, tennis, hockey matches, leagues, news, etc… to create a sporting site/application such as livescore.com
```
Comment (TheLearneer) [inline src/content/apis/sports/LiveScore API.md:3]: Can we please shorten the description a bit?

File: src/content/apis/travel/IRCTC_API.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+name: IRCTC API
+description: An API with various functionalities for Indian railways IRCTC API. We are not affiliated with Indian railways IRCTC in any form, and this is not an official API, but all the results are real-time and accurate.
```
Comment (TheLearneer) [inline src/content/apis/travel/IRCTC_API.md:3]: Can we please shorten the description a bit?

File: src/content/apis/sports/LiveScore API.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+categories:
+  - sports
+features:
+  https: true
+url: https://livescore6.p.rapidapi.com/matches/v2/list-live
```
Comment (TheLearneer) [inline src/content/apis/sports/LiveScore API.md:8]: Does the API have any proper documentation?
The current URL you shared doesn't have any proper info about how to use it.

File: src/content/apis/travel/IRCTC_API.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+categories:
+  - Travel
+features: Get real-time and accurate result.
+  https: true
+url: https://irctc1.p.rapidapi.com/api/v1/searchStation
```
Comment (TheLearneer) [inline src/content/apis/travel/IRCTC_API.md:8]: Does the API have any proper documentation?
The current URL you shared doesn't have any proper info about how to use it.

**Review by TheLearneer — CHANGES_REQUESTED** [review]

File: src/content/apis/travel/IRCTC_API.md (hunk 4 above)
Comment (anupeshverma) [reply src/content/apis/travel/IRCTC_API.md:8]: Should I put the link of documentation also ??

File: src/content/apis/travel/IRCTC_API.md (hunk 2 above)
Comment (anupeshverma) [reply src/content/apis/travel/IRCTC_API.md:3]: Ok I'll do it

File: src/content/apis/travel/IRCTC_API.md (hunk 4 above)
Comment (TheLearneer) [reply src/content/apis/travel/IRCTC_API.md:8]: Instead of this link itself, I would suggest adding the documentation link 🙏

Comment (anupeshverma) [reply src/content/apis/travel/IRCTC_API.md:8]: OK 😊

File: src/content/apis/travel/IRCTC_API.md [hunk 5]
```diff
@@ -0,0 +1,10 @@
+---
+name: IRCTC API
+description: An API with various functionalities for Indian railways IRCTC API.
+categories:
+  - Travel
```
Comment (TheLearneer) [inline src/content/apis/travel/IRCTC_API.md:5]:
```suggestion
  - travel
```

**Review by TheLearneer — APPROVED** [review]
Thank you for your contribution @anupeshverma 🙏

**Pull request merged**